Added a hint to remove static members (#1651)
exercism/csharp: exercises/concept/faceid-2/.docs/hints.md

## 3. Register new identities

> The most performant, and "natural", way to handle a collection of unique values is by using a [hash set][hash-set]. If you encounter issues with the [test runner][exercism-test-runner] failing, try using instance members rather than `static` members in the `Authenticator` class.

Reviewer: ```suggestion
The most performant, and "natural", way to handle a collection of unique values is by using a [hash set][hash-set]. If you encounter issues with the tests passing locally but failing on the website, try using instance members rather than `static` members in the `Authenticator` class.
```

## 4. Prevent invalid identities being authenticated

> Consider the [`HashSet<T>.Contains()`][hash-set-contains] method. If you encounter issues with the [test runner][exercism-test-runner] failing, try using instance members rather than `static` members in the `Authenticator` class.

Reviewer: ```suggestion
Consider the [`HashSet<T>.Contains()`][hash-set-contains] method. If you encounter issues with the tests passing locally but failing on the website, try using instance members rather than `static` members in the `Authenticator` class.
```

Reviewer: @CodeVelo Once this change is applied, we can merge!

This [documentation][reference-equality] addresses the issue of comparing instances as opposed to values.

> ## 6. Potential workaround for failing tests

Reviewer: The number hints should directly correspond to a task. In this case, I would add the hint either to task 3 or 4 (or both), or else maybe to the General tasks lists.

Author: Good to know! See if the changes look ok now.
